addons/account/migrations/12.0.1.1/pre-migration.py

```diff
@@ -117,6 +117,17 @@ def fill_account_invoice_line_sections(cr):
     )
 
 
+def prefill_account_chart_template_transfer_account_prefix(env):
+    """We do this for avoiding a temporary warning on null values."""
+    openupgrade.add_fields(
+        env, [('transfer_account_code_prefix', 'account.chart.template',
+               'account_chart_template', 'char', False, 'account')],
+    )
+    openupgrade.logged_query(
+        env.cr, "UPDATE account_chart_template "
+                "SET transfer_account_code_prefix = 'OUB'")
```

Comment:
```suggestion
                "SET transfer_account_code_prefix = '-OU'")
```
maybe?

Reply:
This is the same than the rest of prefixes in post.